ap_ruby_sdk/lib/ap_ruby_sdk.rb

```diff
@@ -56,10 +56,10 @@ def self.request(method, url, api_key, params={}, headers={})
 
     request_opts = {
         :headers => request_headers(api_key).update(headers),
-        :method => method, :open_timeout => 30,
-        :payload => payload, :url => url, :timeout => 80
+        :method => method,
+        :payload => payload,
+        :url => url
```

Comment:
Think it is useful to leave timeouts, not to get stuck and bombarded with some huge response (just for the case). What are defaults in open_timeouts and timeout?